src/main/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverter.java

```diff
@@ -117,6 +117,19 @@ public class ContentPackage2FeatureModelConverter extends BaseVaultPackageScanne
 
     private IndexManager indexManager = new DefaultIndexManager();
 
+    private RunmodePolicy runmodePolicy = RunmodePolicy.DIRECT_ONLY;
+
+    public enum RunmodePolicy {
```

Comment:
```suggestion
    public enum RunModePolicy {
```

Also in several other places the camel casing should be fixed. It is run mode not runmode.